src/pages/create-or-edit-lesson/CreateOrEditLesson.tsx

```diff
@@ -156,10 +162,7 @@ const CreateOrEditLesson = () => {
   })
 
   useEffect(() => {
-    if (id) {
-      void fetchDataLesson(id)
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
+    if (id) void fetchDataLesson(id)
   }, [])
 
```

Comment:
check this , or add this eslint property if needed: 
// eslint-disable-next-line react-hooks/exhaustive-deps

Reply:
fixed